translations/compfest-11-penyisihan/compfest-11-penyisihan-mengubah-karakter.html

```diff
@@ -0,0 +1,73 @@
+<!-- Title : Altering Character -->
+
+<h3> Description </h3>
+
+<p>
+	In the beginning, Mr. Chanek has a word with $N$ character(s). Mr.Chanek who just wants some fun would like to alter a character in the word into some different characters.
```

Comment:
```suggestion
	In the beginning, Mr. Chanek has a word with $N$ characters. Mr. Chanek who just wants some fun would like to alter a character in the word into some other characters.
```

Kalau kuantitasnya variabel, default ke plural form aja